CMakeLists.txt

```diff
@@ -182,6 +182,12 @@ if (ENABLE_COMPILETIME_FORMAT_CHECK)
     add_compile_definitions(ENABLE_COMPILETIME_FORMAT_CHECK)
 endif()
 
+if("${SERENITY_ARCH}" STREQUAL "aarch64")
+    # FIXME: re-enable this warning
+    add_compile_options(-Wno-type-limits)
+    add_compile_options(-fsigned-char)
```

Comment:
Please modify the GCC spec files instead. Having to inject extra CFLAGS into port builds (how forward-thinking, I know) just to match the system ABI doesn't sound fun.

Reply:
Makes perfect sense. Should be fixed now. I'm keeping the `-W` flag in our CMake config because I guess not every port wants that.